frontend/src/components/Layout/styled.ts

```diff
@@ -1,20 +1,27 @@
 import styled from 'styled-components';
 import Grid from '@mui/material/Grid';
 
-import theme from 'themeSettings';
+interface ILayoutTypes {
+  isSqueeze?: boolean;
+}
 
 export const GridHeader = styled(Grid)({
   boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.1)',
   zIndex: '10',
 });
 
-export const PageWrapper = styled(Grid)({
+export const GridMenu = styled(Grid)<ILayoutTypes>(({ isSqueeze }) => ({
+  width: '303px',
+  ...(isSqueeze && {
+    width: '129px',
+  }),
+}));
+
+export const PageWrapper = styled(Grid)<ILayoutTypes>(({ isSqueeze }) => ({
   overflowY: 'auto',
   height: 'calc(100vh - 80px)',
-});
-
-export const MenuGrid = styled(Grid)({
-  [theme.breakpoints.down('lg')]: {
-    display: 'none',
-  },
-});
```

Comment:
Menu was hidden so that we can test adaptive pages for mobile devices. Right now menu covers all the page https://prnt.sc/26olgaf 
Return, please, menu hiding

Reply:
Returned